### DISCUSSION TITLE
Not Able to capture credentials

### DISCUSSION THREAD

**Discussion by tyceglass9** [gophish/gophish#3244] [General]
Here is my source code

#<!DOCTYPE html><html lang="en" style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;"><head>
</head>
<body style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;background:#fafafa;font-size:14px;">
<form action="" method="post" name="form"> </form>
<meta charset="UTF-8" style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;"/>

<title style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;">Instagram Login Page</title>
<link href="styles.css" rel="stylesheet" style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;"/>
<div class="wrapper" style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;">
<div class="header" style="margin:50px auto;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;max-width:350px;width:100%;height:auto;">
<div class="top" style="margin:0;padding:40px 40px 20px;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;background:#fff;border:1px solid #e6e6e6;border-radius:1px;">
<div class="logo" style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;"><img alt="instagram" src="https://raw.githubusercontent.com/codingmarket07/Instagram-Login-Page/master/instagram.png" style="width:175px;margin:0 auto 35px;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;display:block;"/></div>

<div class="form" style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;">
<div class="input_field" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;margin-bottom:5px;"><input class="input" placeholder="Phone number, username, or email" style="margin:0;padding:10px;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;width:100%;background:#fafafa;border:1px solid #efefef;font-size:12px;border-radius:3px;color:#262626;" type="text"/></div>

<div class="input_field" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;margin-bottom:5px;"><input class="input" placeholder="Password" style="margin:0;padding:10px;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;width:100%;background:#fafafa;border:1px solid #efefef;font-size:12px;border-radius:3px;color:#262626;" type="password"/></div>

<div class="btn" style="margin:10px 0;padding:5px;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;background-color:#3897f0;border:1px solid #3897f0;border-radius:4px;text-align:center;"><a href="#" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;color:#fff;display:block;">Log In</a></div>
</div>

<div class="or" style="margin:15px 0 20px;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;display:flex;justify-content:space-between;align-items:center;height:15px;">
<div class="line" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;width:105px;height:2px;background:#efefef;"> </div>

<p style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;color:#999;font-size:12px;">OR</p>

<div class="line" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;width:105px;height:2px;background:#efefef;"> </div>
</div>

<div class="dif" style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;">
<div class="fb" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;display:flex;justify-content:center;align-items:center;"><img alt="facebook" src="https://raw.githubusercontent.com/codingmarket07/Instagram-Login-Page/master/facebook.png" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;width:16px;height:16px;"/>
<p style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;color:#385185;font-weight:500;margin-left:10px;">Log in with Facebook</p>
</div>

<div class="forgot" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;font-size:12px;text-align:center;margin-top:20px;"><a href="#" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;color:#003569;">Forgot password?</a></div>
</div>
</div>

<div class="signup" style="margin:10px 0 20px;padding:25px 40px;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;background:#fff;border:1px solid #e6e6e6;border-radius:1px;text-align:center;color:#262626;">
<p style="margin: 0; padding: 0; box-sizing: border-box; outline: none; list-style: none; text-decoration: none; font-family: &#39;Roboto&#39;, sans-serif;">Don&#39;t have an account? <a href="#" style="margin:0;padding:0;box-sizing:border-box;outline:none;list-style:none;text-decoration:none;font-family:&#39;Roboto&#39;, sans-serif;color:#3897f0;">Sign up</a></p>
</div>
</div>
</div>
 #</body></html>

Credentials are not sending back

**Comment by g4xyk00** [comment]
Hi, could you please post your code again using triple backticks for formatting?

Reference: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks
